src/iam/login/login.service.spec.ts

```diff
@@ -94,26 +105,19 @@ describe('LoginService', () => {
       expect(await loginService.login(loginDto)).toEqual(userLogin);
     });
 
-    it('should return an exception if login fails', async () => {
-      jest
-        .spyOn(loginService, 'login')
-        .mockRejectedValue(new UnauthorizedException('User does not exists'));
-
-      await expect(loginService.login(loginDto)).rejects.toThrow(
-        new UnauthorizedException('User does not exists'),
-      );
-    });
-
     it('should return an exception if wrong password', async () => {
-      usersService.findByEmail = jest.fn().mockRejectedValueOnce(null);
-      hashingService.compare = jest.fn().mockReturnValue(false);
+      usersService.findByEmail = jest.fn().mockResolvedValueOnce(oneUser);
+      hashingService.compare = jest.fn().mockResolvedValueOnce(false);
       await expect(
-        hashingService.compare('password', 'not a correct password'),
-      ).toBe(false);
```

Comment:
You shouldn't be testing the method you mock directly. That's like asserting that jest is doing what it should be, which should already be covered by the library's tests